.github/workflows/create_publish_artifacts.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and Package
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Runs the workflow at midnight every day
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  # build and run for RISCV
+  run_riscv_m1_nightly_package:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+       # installs tools, ninja, installs llvm and sets up sccahe
+      - name: setup ubuntu
+        uses:  ./.github/actions/setup_ubuntu_build
+        with:
+          llvm_version: 17
+          llvm_build_type: RelAssert
+
+      - name: setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+
+      - name: build riscv M1
+        uses: ./.github/actions/do_build_ock/do_build_m1
+
+      - name: Download Daily Release
+        run: |
+          wget "https://github.com/intel/llvm/releases/download/nightly-2023-12-18/sycl_linux.tar.gz"
+          mkdir linux_nightly_release
+          tar -xzf sycl_linux.tar.gz -C linux_nightly_release
+
+      - name: Clone oneAPI-samples
+        run: git clone https://github.com/oneapi-src/oneAPI-samples.git
+
+      - name: Setup environment and test Vector Add
+        run: |
+          export ONEAPI_ROOT=$(pwd)/linux_nightly_release
+          export PATH=$(pwd)/linux_nightly_release/bin:$PATH
+          export LD_LIBRARY_PATH=$(pwd)/linux_nightly_release/lib:$(pwd)/build/lib:$LD_LIBRARY_PATH
+
+          $(pwd)/linux_nightly_release/bin/clang++ -fsycl $(pwd)/oneAPI-samples/DirectProgramming/C++SYCL/DenseLinearAlgebra/vector-add/src/vector-add-buffers.cpp -o vect
+          CA_HAL_DEBUG=1 OCL_ICD_FILENAMES=$(pwd)/build/lib/libCL.so ONEAPI_DEVICE_SELECTOR=opencl::acc  SYCL_CONFIG_FILE_NAME="" ./vect
```

Comment:
Should this be `opencl:acc` or is `opencl::acc` intended? I always use the former.